app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -206,12 +207,28 @@ class DefaultBrowserToolbarController(
                         .show()
                 }
             }
-            ToolbarMenu.Item.AddToHomeScreen, ToolbarMenu.Item.InstallToHomeScreen -> {
+            ToolbarMenu.Item.AddToHomeScreen -> {
                 activity.settings().installPwaOpened = true
                 MainScope().launch {
                     with(activity.components.useCases.webAppUseCases) {
                         if (isInstallable()) {
                             addToHomescreen()
+                            activity.metrics.track(Event.ProgressiveWebAppAddToHomescreenTap)
```

Comment:
Maybe this should be named AddToHomescreenTap, as this system is just for bookmark shortcuts instead of PWAs.

Reply:
Should this be emitted before or after the `addToHomescreen` suspend function?

Reply:
Ahhh this is the wrong place to put it then. This telemetry is only for PWAs.